README.md

```diff
@@ -156,14 +156,16 @@ To **support the password or the client credentials flow, you must provide your
 
 ```csharp
 [HttpPost("~/connect/token")]
+[Produces("application/json")]
```

Comment:
Is this line necessary?

I get application/json content type in the response headers when i sent that header.